<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>44</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>345</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="DbComboBox" name="comboBoxDBType"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>DB connection:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pathDBEdit">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>:memory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushPathDbButton">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select file to containing Data base&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/folder.png</normaloff>
           <activeon>icons/folder-open.png</activeon>icons/folder.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>71</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPlainTextEdit" name="SQLTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="SelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableSQLResult">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">my_widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxDBType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onCurrentComboDbChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushPathDbButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPushPathDb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPushExecuteSQL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPushExecuteSQL()</slot>
  <slot>onPushPathDb()</slot>
  <slot>onCurrentComboDbChange()</slot>
 </slots>
</ui>
